composer.json

```
@@ -19,7 +19,7 @@
     "require-dev": {
         "ext-pdo": "*",
         "contao/core-bundle": "^4.9",
-        "doctrine/dbal": "^2.10",
+        "doctrine/dbal": "^2.11 || 3.0",
```

```suggestion
        "doctrine/dbal": "^2.11 || ^3.0",
```

Should this be `^3.0`

---

Fixed in 70d69cf59ef0725474794136d8df91f96c05c625